scripts/password_less_sudo.sh

```
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Check if the script is run as root
+if [ "$(id -u)" -ne 0; then
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)